apps/src/studio/skins.js

```diff
@@ -329,6 +329,14 @@ function loadHoc2015(skin, assetUrl) {
     };
   });
 
+  skin.bot1.movementAudio = [
+    { begin: 'roll_begin', loop: 'roll_loop', end: 'roll_end' }
+  ];
+  skin.bot2.movementAudio = [
+    { loop: 'bot2_move_loop', end: 'bot2_move_end' },
+    { loop: 'bot2_move2_loop', end: 'bot2_move2_end' }
+  ];
```

Comment:
We switch the set of movement sounds for the current sprite whenever we set the sprite.  Also, note that you can skip the "begin" sound - it's actually safe to omit any of "begin" "loop" or "end" and the `ThreeSliceAudio` will just skip that part.